assignments/B-03-84/models/user.js

```diff
@@ -0,0 +1,83 @@
+const mongoose = require('mongoose');
+const schema = mongoose.Schema;
```

Comment:
This schema is wrong because  we are not getting this much info. Keep this schema simple we will enhance it later